pkgs/applications/graphics/write_stylus/default.nix

```diff
@@ -36,8 +29,11 @@ mkDerivation rec {
     # symlink the binary to bin/
     ln -s $out/Write/Write $out/bin/Write
 
+    # Create desktop item
     mkdir -p $out/share/applications
     ln -s ${desktopItem}/share/applications/* $out/share/applications/
+    mkdir -p $out/share/icons
+    ln -s $out/Write/Write144x144.png $out/share/icons/write.png
```

Comment:
Thanks for making the changes! I am a bit worried that `write.png` is very generic and may clash with another package when installed into a profile. Perps you could change this to something like `write_stylus.png`? The changes look good otherwise.

Reply:
You're right, I changed it.